packages/wrangler/src/__tests__/patch-config.test.ts

```diff
@@ -697,6 +697,35 @@ describe("experimental_patchConfig()", () => {
 				`);
 			});
 		});
+
+		it("should not error if a `null` is passed in", () => {
```

Comment:
This is completely unnecessary (especially given the other changes in this PR), but currently if `patchConfig` finds a `null` it errors with a very unhelpful error message:
<img width="452" height="37" alt="Screenshot 2025-11-14 at 18 03 02" src="https://github.com/user-attachments/assets/41fed2a4-8915-4dc2-83bd-298a87b0171a" />

So I think that allow it to instead apply the (incorrect) `null` is better

(Alternatively we could provide a more helpful error message saying that `null` values are not expected?)